docs/Makefile

```diff
@@ -45,12 +45,12 @@ lint: lint-fix
 # - .markdownlintignore holds the configuration for files to be ignored
 # - .markdownlint.yaml contains the rules for markdownfiles
 # renovate: datasource=docker depName=davidanson/markdownlint-cli2-rules
-MDL_DOCKER_VERSION := v0.12.0
-MDL_CMD := docker run -v $(ROOT_DIR)../:/workdir --rm 
+MDL_DOCKER_VERSION := v0.12.1
+MDL_CMD := docker run -v $(ROOT_DIR)../:/workdir --rm
 
 .PHONY: markdownlint markdownlint-fix
 markdownlint:
-	$(MDL_CMD) davidanson/markdownlint-cli2-rules:${MDL_DOCKER_VERSION} "**/*.md" 
+	$(MDL_CMD) davidanson/markdownlint-cli2-rules:${MDL_DOCKER_VERSION} "**/*.md"
 
 markdownlint-fix:
-	$(MDL_CMD) --entrypoint="markdownlint-cli2-fix" davidanson/markdownlint-cli2-rules:${MDL_DOCKER_VERSION} "**/*.md" 
+	$(MDL_CMD) davidanson/markdownlint-cli2-rules:${MDL_DOCKER_VERSION} --fix "**/*.md"
```

Comment:
The command did not actually fix the issues :/

make markdownlint-fix
docker run -v /Users/ondrej.dubaj/KEPTN/lifecycle-toolkit/docs-new/../:/workdir --rm davidanson/markdownlint-cli2-rules:v0.12.1 --fix "**/*.md"
markdownlint-cli2 v0.12.1 (markdownlint v0.33.0)
Finding: **/*.md !**/CHANGELOG.md !node_modules !docs/tmp !readme-generator-for-helm
Linting: 221 file(s)
[Error: EACCES: permission denied, open '/workdir/docs-new/docs/installation/configuration/cert-manager.md'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'open',
  path: '/workdir/docs-new/docs/installation/configuration/cert-manager.md'
}
make: *** [markdownlint-fix] Error 2

Something wrong with the permissions...

Reply:
Most likely is due to this line: https://github.com/DavidAnson/markdownlint-cli2/blob/main/docker/Dockerfile#L20
Since the local files don't belong to the user `user`, there is a permission error. Try running it w/ `sudo` permissions 🤔

Reply:
interesting.. for me, it works fine and it's able to fix things